test/unit/test_utils.py

```diff
@@ -70,3 +72,31 @@ def test_sha256_streaming(size):
     actual_digest = utils.sha256_streaming(io.BytesIO(buf))
 
     assert expected_digest == actual_digest
+
+
+def test_load_pem_public_key_format():
+    keybytes = b"-----BEGIN PUBLIC KEY-----\n" b"bleh\n" b"-----END PUBLIC KEY-----"
+    with pytest.raises(
+        utils.InvalidKey, match="could not load PEM-formatted public key"
+    ):
+        CTKeyring([keybytes])
```

Comment:
Rather than testing through `CTKeyring`, maybe test `load_pem_public_key` directly since this test is under the `utils` unit?